gui/gui-manager.cpp

```diff
@@ -517,7 +517,7 @@ void GuiManager::processEvent(const Common::Event &event, Dialog *const activeDi
 	Common::Point mouse(event.mouse.x - activeDialog->_x, event.mouse.y - activeDialog->_y);
 	switch (event.type) {
 	case Common::EVENT_KEYDOWN:
-		activeDialog->handleKeyDown(event.kbd);
+		activeDialog->handleKeyDown(event.kbd, event.synthetic);
```

Comment:
I think it's best adding a TODO here which marks that using "synthetic" is not correct here since more than just keyboard repeat events are synthetic (it's "keyboard repeat => synthetic == true" not the other way round).